Formula/mpv.rb

```diff
@@ -18,9 +18,8 @@ class Mpv < Formula
 
   depends_on "docutils" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.9" => :build
+  depends_on "python@3.10" => :build
   depends_on xcode: :build
-
   depends_on "ffmpeg@4"
   depends_on "jpeg"
```

Comment:
Would like to migrate `jpeg --> jpeg-turbo` and `luajit-openresty --> luajit`. Still blocked by Big Sur Xcode issue though.